src/viem/saga.ts

```diff
@@ -282,7 +281,6 @@ export function* sendAndMonitorTransaction({
       ...commonTxAnalyticsProps,
       txHash: hash,
     })
-    yield* put(addHashToStandbyTransaction(context.id, hash))
```

Comment:
Ah damn this was a valid use case for it. 
Until we refactor things.
Sorry I read that too fast.